addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -795,7 +753,7 @@ export class Rtc {
         this.clear();
         this.state.logs.clear();
         this.state.channel = channel;
-        this.onThreadUpdate(this.state.channel, { rtcSessions, invitedMembers });
```

Comment:
Note: removed because was dead code, but important to remove because original `update()` would clear `invitedMembers` fields.

I'm not entirely sure this is fine or not. To play safe, I prefer to remove what was dead code